core/controllers/contributor_dashboard_admin_test.py

```diff
@@ -763,3 +764,843 @@ def test_stats_data_not_calculated_without_topic_id_and_language_code(
+
+    def test_get_translation_submitter_stats_for_last_activity_filter(self) -> None: # pylint: disable=line-too-long
+
+        self.login(self.CONTRIBUTOR_EMAIL)
+
+        # Test with num_days_since_last_activity filter and pagination.
+        response = self.get_json(
+            '/contributor-dashboard-admin-stats/translation/submission', {
+                'page_size': 4,
+                'offset': 1,
+                'language_code': self.SUGGESTION_LANGUAGE_CODE,
+                'num_days_since_last_activity': 120,
+                'topic_ids': []
+            })
+
+        self.assertEqual(
+            len(response['stats']),
+            3
+        )
+        self.assertEqual(
+            response['stats'][0]['contributor_id'],
+            'user3'
+        )
+        self.assertEqual(
+            response['stats'][1]['contributor_id'],
+            'user2'
+        )
+        self.assertEqual(
+            response['next_offset'],
+            4
+        )
+        self.assertEqual(
+            response['more'],
+            False
+        )
+        self.logout()
+
+    def test_get_translation_reviewer_stats_for_pagination(self) -> None:
+
+        self.login(self.CONTRIBUTOR_EMAIL)
+
+        # Test with language filter and pagination.
+        response = self.get_json(
+            '/contributor-dashboard-admin-stats/translation/review', {
+                'page_size': 2,
+                'offset': 1,
+                'language_code': self.SUGGESTION_LANGUAGE_CODE,
+                'topic_ids': []
+            })
+
+        self.assertEqual(
+            len(response['stats']),
+            2
+        )
+        self.assertEqual(
+            response['stats'][0]['contributor_id'],
+            'user3'
+        )
+        self.assertEqual(
+            response['next_offset'],
+            3
+        )
+        self.assertEqual(
+            response['more'],
+            True
+        )
+        self.logout()
+
+    def test_get_translation_reviewer_stats_for_sorting(self) -> None:
+
+        self.login(self.CONTRIBUTOR_EMAIL)
+
+        # Test with sorting and pagination.
+        response = self.get_json(
+            '/contributor-dashboard-admin-stats/translation/review', {
+                'page_size': 2,
+                'offset': 1,
+                'language_code': self.SUGGESTION_LANGUAGE_CODE,
+                'sort_by': 'IncreasingLastActivity',
+                'topic_ids': []
+            })
+
+        self.assertEqual(
+            len(response['stats']),
+            2
+        )
+        self.assertEqual(
+            response['stats'][0]['contributor_id'],
+            'user2'
+        )
+        self.assertEqual(
+            response['next_offset'],
+            3
+        )
+        self.assertEqual(
+            response['more'],
+            True
+        )
+        self.logout()
+
+    def test_get_translation_reviewer_stats_for_last_activity_filter(self) -> None: # pylint: disable=line-too-long
```

Comment:
Don't think the lint exception is necessary if we indent properly.
```suggestion
    def test_get_translation_reviewer_stats_for_last_activity_filter(
      self
    ) -> None:
```

Reply:
Yeah, Thanks for the suggestion.